src/theme.js

```diff
@@ -135,13 +135,20 @@ export const radius = '2px'
 
 export const maxContainerWidth = '1280px'
 
-const shadowColor = 'rgba(0,0,0,0.08)'
-export const boxShadows = [
-  `0 0 4px 0 ${shadowColor}`,
-  `0 8px 8px 0 ${shadowColor}`,
-  `0 16px 16px 0 ${shadowColor}`,
-  `0 32px 32px 0 ${shadowColor}`
-]
+// boxShadows
+const boxShadowSm = '0 0 2px 0 rgba(0,0,0,.08),0 1px 4px 0 rgba(0,0,0,.16)'
+const boxShadowMd = '0 0 2px 0 rgba(0,0,0,.08),0 2px 8px 0 rgba(0,0,0,.16)'
+const boxShadowLg = '0 0 2px 0 rgba(0,0,0,.08),0 4px 16px 0 rgba(0,0,0,.16)'
+const boxShadowXl = '0 0 2px 0 rgba(0,0,0,.08),0 8px 32px 0 rgba(0,0,0,.16)'
+
+const boxShadows = {
```

Comment:
Could this be a breaking change for anyone who was using boxShadows as `boxShadows[1]`?

Reply:
That's a good question - I didn't think of applications outside of the `Card` and `Tooltip` components. These new boxShadow values do not layer shadows like the old values did so that's why I changed the configuration.